webapp/src/monacoSnippets.ts

```diff
@@ -617,6 +617,25 @@ export function blockIdMap() {
     return _blockIdMap;
 }
 
+export function tsSnippetToPySnippet(param: string): string | undefined {
+    // Keep this unified with pxtcompiler/languageservice.ts
+    const keywords: pxt.Map<string> = {
+        "true": "True",
+        "false": "False",
+        "null": "None"
+    }
+    const key = keywords[param];
+    if (key) {
+        return key
+    }
+    if (param.includes(".")) {
+        // Python enums are all caps
+        const match = /(.+)\.(.+)/.exec(param);
```

Comment:
```suggestion
        const match = /(.+)\.([^.]+)$/.exec(param);
```

maybe to only grab the final? we have some enums (e.g. `sprites.Flag.Ghost`) in namespaces but I feel like I remember them having trouble with blocks so maybe it's not actually an issue (in this case we have a trimmed version of the enum at the top level anyways)

Reply:
ooooh yeah I'll do that and test